fastchat/serve/model_worker.py

```diff
@@ -101,6 +100,10 @@ def __init__(
             self.init_heart_beat()
 
     def generate_stream_gate(self, params):
+        if self.device == "npu":
```

Comment:
Move this to the initialization function (`__init__``)

Reply:
move this to __init__ can't fix this bug
it happens when curl
https://github.com/lm-sys/FastChat/blob/e67b21dcbde91a5ad9740b081c59c433093f54da/fastchat/serve/openai_api_server.py#L507